.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "parallel_algorithms/wg21"]
+	path = parallel_algorithms/wg21
+	url = https://github.com/mpark/wg21.git
```

Comment:
First, I would add the submodule as a separate patch to the main branch instead of this paper patch.

Second, it's up to you of course but why not adding it into the root instead of `parallel_algorithms`?

Reply:
Fixed by removing